server/src/main/java/org/opensearch/transport/TransportService.java

```diff
@@ -858,6 +866,8 @@ public final <T extends TransportResponse> void sendRequest(
     ) {
         try {
             logger.debug("Action: " + action);
+            final SpanScope spanScope = tracer.startSpan(action, populateAttributes(action, connection));
```

Comment:
Referring to the previous comment, please think about descriptive span name, for example: action  + " " + <remote host>